torch/_prims/__init__.py

```diff
@@ -206,13 +206,16 @@
 # the present mode will also deallocate. FakeTensorMode holds onto
 # tensors that are converted to Meta so we don't want to persist it
 # longer than necessary.x
+#
+# TODO: This seems dodgy, figure out how to not do this
 prim_fake_mode_ref = None
```

Comment:
yeaaaa this was adding kind of hastily when I was rebasing adding Modes and kept running into prim errors.... I think with composite compliance this shouldnt be necessary but at the time there were a lot of tensor constructors that didn't use `new_empty` etc and friends.. tbh was surprised having run into more errors with it haha. 

I can take a look at removing

Reply:
The general model is that the mode should take care of ensure factory function calls (which would otherwise not be composite compliant) get transformed into meta/fake tensors, so you don't have to do it explicitly. It would be good to understand why this is not working